test/unitTests/documentSymbol/snapshot.test.ts

```diff
@@ -27,12 +27,19 @@ test.each(
   expect(symbols).toMatchSnapshot();
 });
 test('Testing workspace symbol', () => {
-  const symbols = workspaceSymbol({query: ''}, projectParser, [])?.map(symbol => ({
+  const symbols = workspaceSymbol({ query: '' }, projectParser, [])?.map(symbol => ({
     ...symbol,
     location: {
       ...symbol.location,
       uri: symbol.location.uri.replace(pathToFileURL(__dirname).toString(), 'file:///c/dummy/')
     }
   }));
+  symbols?.sort((a, b) => {
+    const lineDifference = b.location.range.start.line - a.location.range.start.line;
+    if (lineDifference !== 0) {
+      return lineDifference;
+    }
+    return b.location.range.start.character - a.location.range.start.character;
+  });
```

Comment:
I assume this was just done to preserve the snapshot for the future and there were no actual changes in the snapshot?

Reply:
Yes there was some weird race condition type thing, where sometimes the snapshot matches sometime not.